### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by huangyz0918**
Please have a review @calpa .
Besides, should we add a link to the `CONTRIBUTING.MD` in the `README.MD` ??

### CODE REVIEW

**Comment by calpa** [comment]
The travis-ci don't expose API_KEY in the pull request, #40 is fixing this issue.

**Comment by huangyz0918** [comment]
> add a link of 'bug report'.

Do you mean adding the `CONTRIBUTING.MD` link in the `README.MD` ??

**Comment by calpa** [comment]
There is a `bug report` wordings in CONTRIBUTING.MD, so add the link to issue page would be better.

And, you may link the README.md to CONTRIBUTING.MD.

**Comment by huangyz0918** [comment]
Okey I got it. 
Maybe we can use Chinese to each other, this would be more efficient :)

现在可以review 了 @calpa

File: README.md [hunk 1]
```diff
@@ -140,6 +140,10 @@ There are two configurable headers, [Homepage](https://calpa.me) and [tags page]
 - `npm run reset` to clear the local cache
 - Check [GatsbyJS Debugging Docs](https://www.gatsbyjs.org/docs/debugging-html-builds/)
 
+## Contributing
+
+Please checkout [here](.github/CONTRIBUTING.md) for more information.
```
Comment (calpa) [inline README.md]: Please read [CONTRIBUTING.md](.github/CONTRIBUTING.md) for more information.

**Comment by calpa** [comment]
@huangyz0918 Thank you for your contribution, and your pull request is merged. If you like this project, please don't hesitate to star it.

**Comment by huangyz0918** [comment]
Okey, nice to working with you.
The issue #43 can be closed.

**Pull request merged**